psychopy_pixx/__init__.py

```diff
@@ -1 +1,6 @@
- 
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from .components.responsepixx import ResponsepixxComponent
```

Comment:
These components/routines should not be exposed to the top-level module (psychopy_pixx). 
You might remove these lines from here